atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -57,8 +57,52 @@ def get_value(workflow, name, fallback):
         raise
 
 
-def get_koji(workflow, fallback=NO_FALLBACK):
-    koji_map = get_value(workflow, 'koji', fallback)
+def add_koji_map_in_workflow(workflow, hub_url, root_url=None, reserve_build=None,
```

Comment:
why and what is this? 

this is used just in tests to set koji values to reactor config,
and in tests we usually already passing dict to ReactorConfig, so passing it to function which will set it isn't useful

Reply:
:eyes: yup mocking function should be in tests.

@rcerven this mocking has been introduced, because koji attribute is mandatory now, and tests we failing with missing `koji` config key

Reply:
I added this function because some plugins' tests do not configure koji config as usual and I need to manually add all configs to each internal test. So I use this function instead of creating configuration dicts every time.

Reply:
This function has been moved to tests/util.py